src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessorImpl.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import io.vavr.Value;
+import io.vavr.control.Validation;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+
+import static io.vavr.control.Validation.invalid;
+import static io.vavr.control.Validation.valid;
+import static java.lang.String.format;
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.reform.ccd.client.model.CallbackTypes.ABOUT_TO_SUBMIT;
+
+@Service
+public class CallbackProcessorImpl implements CallbackProcessor {
```

Comment:
Callback processor I would imagine would retrieve the case and update it . Are you planing to do that later as a separate PR? Not sure why we need validation ? Could you please list down a scenario where you would get validation error ?

Reply:
If there are errors they will happen.
Im using the TDD approach to this problem and doing the slices everyone wants of the smallest bit.
This is a work in progress and probably have 12 or so PR's